booking-app/src/main/java/org/motechproject/bookingapp/helper/DtoLookupHelper.java

```diff
@@ -49,6 +55,59 @@ public static BookingGridSettings changeLookupForPrimeVaccinationSchedule(Bookin
         return settings;
     }
 
+    public static BookingGridSettings changeLookupForScreening(BookingGridSettings settings) throws IOException {
+        Map<String, Object> fieldsMap = new HashMap<>();
+        DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyy-MM-dd");
+
+        DateFilter defaultDateFilter = DateFilter.TODAY;
+
+        if (StringUtils.isBlank(settings.getFields())) {
+            settings.setFields("{}");
+        }
+
+        if (StringUtils.isBlank(settings.getLookup())) {
+            settings.setLookup("Find By Date");
+        } else {
+            String fields = settings.getFields();
+            fieldsMap = OBJECT_MAPPER.readValue(fields, new TypeReference<HashMap>() {});
+            if (settings.getLookup().equals("Find By Volunteer Name")) {
+                settings.setLookup("Find By Date And Volunteer Name");
+            } else {
+                settings.setLookup("Find By Date And Clinic Location");
+            }
+        }
+        if (settings.getDateFilter() != null) {
```

Comment:
all code should be in this condition, because right now if you not select use filter checkbox default filter will be added